oracle/Assets/Scripts/GraphComponent.cs

```diff
@@ -80,7 +106,7 @@ virtual protected void Start()
     /// <summary>
     /// Called every frame.
     /// </summary>
-    virtual protected void Update()
+    virtual protected void LateUpdate()
```

Comment:
what does this do differently?

Reply:
I was experimenting and forgot to change it back. Unity has different methods for different times during the frame and was wondering if changing it would give a better feel.